flepimop/gempyor_pkg/src/gempyor/parameters.py

```diff
@@ -134,14 +134,16 @@ def __init__(
                     print("loaded dates:", df.index)
                     raise ValueError(
                         f"Issue loading file '{fn_name}' for parameter '{pn}': " 
-                        f"The 'date' entries of the provided file do not include all the days specified to be modeled by the config."
+                        f"Provided file dates span '{str(df.index[0])}' to '{str(df.index[-1])}', "
+                        f"but the config dates span '{ti}' to '{tf}'."
```

Comment:
I think precises error messages help a lot, i know there is a discussion in keeping them concise and all, but if we look at e.g https://iddynamicsjhu.slack.com/archives/C07PBSVRJ05/p1729087436855849, it is clear that in a context where the person generating the config is often different from the person running flepiMoP, the more informative the message is the faster is the communication

Reply:
e.g in the linked error, the dates are the same but some dates are duplicated. If we want to keep short error message, we could differenciate the two cases:
1. the end-start dates are different
2. the start-end dates are the same but number of elements is not the same.

Reply:
I can't follow the slack link to see what happened there, but perhaps fixing these exceptions to catch the right thing this is more of an issue for GH-277 then? Can just put the number of days back in the meantime. There is already a comment https://github.com/HopkinsIDD/flepiMoP/issues/276#issuecomment-2272119531 discussing this set of errors.

Reply:
agree on keeping it that way here, and the need for specific errors. In your linked you have identified what has been needed in this slack message (number of date vs just end and start date). adding a screenshot in case: 
<img width="1453" alt="Screenshot 2024-10-21 at 11 27 38" src="https://github.com/user-attachments/assets/987d2e50-a536-4f1b-9a54-44d3199d5749">

Reply:
Heard. Returning the error message to the original level of verbosity!